recipes-kernel/linux/files/patches-5.10/0105-arm64-dts-ti-iot2050-modify-the-usb-related-dts-conf.patch

```diff
@@ -0,0 +1,38 @@
+From ccb066ecd608cfea2990e05b260a05501287539a Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Sun, 26 Sep 2021 10:54:21 +0800
+Subject: [PATCH 105/105] arm64:dts:ti:iot2050:modify the usb related dts
+ configure
+
+As there are some incorrect dts configurations for usb port.
+it needs to re-correct it.
```

Comment:
This provides no information. Can you please describe the change?

Also, in general, please format the patch according to other patches (e.g. add spaces after the colons above). In this case, we will merge the delta in to the patch that introduces the file anyway, but if that patch really went on top, it would require a correct style.